geowebcache/pom.xml

```diff
@@ -49,6 +49,8 @@
     <qa>false</qa>
     <lint>deprecation</lint>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <jclouds.version>2.2.0</jclouds.version>
+    <mockito.version>3.2.4</mockito.version>
```

Comment:
Hmm... probably best not to do this. Mockito version is already used in other modules and managed in this pom, search for mockito-all in this and other poms. However it's an older version of mockito.
If the swift module is doing its own thing best if it's kept private.